packages/libs/wdk-client/src/Views/Answer/AnswerFilter.jsx

```diff
@@ -46,7 +46,8 @@ class AnswerFilter extends React.Component {
     }
   }
 
-  toggleFilterFieldSelector() {
+  toggleFilterFieldSelector(e) {
+    e.stopPropagation();
```

Comment:
This will cause an error because this method is passed to `AnswerFilterSelector` and called it with no arguments.

Something different to try is adding also checking if `this.props.open === true` to the condition on this line: https://github.com/VEuPathDB/web-monorepo/blob/84d2f4204b094dec181456ea7f721bebab011f2f/packages/libs/wdk-client/src/Views/Answer/AnswerFilterSelector.jsx#L37

Reply:
I ended up moving the `stopPropagation` call to a click handler in the button itself. Adding `this.props.open` to that condition didn't prevent the undesired behavior. By the time the code reaches that condition, `this.props.open` still registers as true.

Reply:
Ah, well, this looks like a great solution!